CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 3.6.0 (YYYY-MM-DD)
 
+### Breaking Changes
+
+* [ObjectServer] Read-only is not supported by synced Realm. "SyncConfigration.readOnly()" will always throw `UnsupportedOperationException`.
```

Comment:
Well, this has to be changed here. Is it acceptable for current master? or should those just go to master-4.0?